sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_customers_history_v1/metadata.yaml

```diff
@@ -8,7 +8,7 @@ description: |-
 
   This table's schema closely mirrors Stripe's customers API (https://stripe.com/docs/api/customers/object).
   Fields which Fivetran doesn't sync and fields we've specifically chosen not to include (e.g. PII) have been omitted.
-  In particular, all address fields except country have been omitted because Firefox Account user IDs are present and this is in a Mozilla-confidential dataset.
+  In particular, postal codes have been omitted from address because Firefox Account user IDs are present and this is in a Mozilla-confidential dataset.
```

Comment:
Is this line still relevant?  From what I can tell, the source tables are moz-confidential

Reply:
Ah ok I saw that the redash service account had access to `stripe_external` so I assumed it was accessible from stmo but that role doesn't include data viewer permissions.  I also didn't see that data platform group has special access